specification/sdk-tracing.md

```diff
@@ -260,6 +260,52 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter. These spans are called _unfinished_ spans.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
+have been started more than `reportIntervalMillis` ago, are exported.
+
+**Configurable parameters:**
+
+* `exporter` - the exporter where the spans are pushed.
+* `maxWatchedSpans` - the maximum number of unfinished spans to be watched.
+  While reached, no new spans will be accepted. The default value is `2048`.
+* `reportIntervalMillis` - the delay interval in milliseconds between two
+  consecutive exports. The default value is `5000`.
+* `exporterTimeoutMillis` - how long the export can run before it is cancelled.
+  The default value is `30000`.
+* `maxExportBatchSize` - the maximum batch size of every export. It must be
+  smaller or equal to `maxWatchedSpans`. The default value is `512`.
+* `exportEndedSpans` (boolean) - if set, spans are also exported once they are
+  ended.
+  The default value is `false`.
+* `spanDurationTimeoutMillis` - On each report interval, spans older than
+  `spanDurationTimeoutMillis` which have not yet ended will be dropped from
+  the watch list.
+  The default is `60000`.
+  (_only applies to languages where weak references are not supported or
+  suitable_)
+
+In order to avoid multiple exports potentially being blocked on connection issues or
+when `reportIntervalMillis` is set inappropriately, exports should *not* be performed
+in parallel. If a previous export is still in progress on the next
+`reportIntervalMillis`, the current export should be skipped.
```

Comment:
"on the next reportIntervalMillis" sounds strange to me. I suggest "If an export would normally happen while another export by this SpanProcessor is still running, the export is skipped."